pkgs/by-name/an/ankama-launcher/package.nix

```
@@ -0,0 +1,44 @@
+{
+  appimageTools,
+  fetchurl,
+  lib,
+}:
+let
+  pname = "ankama-launcher";
+  version = "3.12.17";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://launcher.cdn.ankama.com/installers/production/Ankama%20Launcher-Setup-x86_64.AppImage";
+    hash = "sha256-aD5YYcye/WPpb0Kir55qCL+RGbK8ZSdmi4NKmqKkW/Y=";
```

The corresponding hash should be the following:
```suggestion
    hash = "sha256-ssCjFrQYI/pxnsjcy1xLSpuGGy0NGMOcNP9RiNDhE/w=";
```